src/assets/locales/en.js

```diff
@@ -285,6 +285,20 @@ export default {
       enter: "Go Fullscreen",
       exit: "Exit Fullscreen",
     },
+    scorecard: {
+      greeting: "Hooray! Congratulations on completing the Plio!",
```

Comment:
Plio: company
plio: interactive video

```suggestion
      greeting: "Hooray! Congratulations on completing the plio!",
```